website/project/model.py

```diff
@@ -75,6 +75,7 @@ def has_anonymous_link(node, auth):
 contributor_added = signals.signal('contributor-added')
 unreg_contributor_added = signals.signal('unreg-contributor-added')
 write_permissions_revoked = signals.signal('write-permissions-revoked')
+wiki_updated = signals.signal('wiki-updated')
```

Comment:
Any new signals should be added to `website.signals.ALL_SIGNALS`. This allows them to be captured by our `CaptureSignals` testing utility and allows any signal to be imported from a single module.